src/ecs/components.h

```diff
@@ -31,6 +32,7 @@
 #define LAYER_NONE ((u64)0)
 #define LAYER_TERRAIN ((u64)1 << 0)
 #define LAYER_LETHAL ((u64)1 << 1)
+#define LAYER_INTERACTABLE ((u64)1 << 1)
```

Comment:
Do we need this new layer? Not sure how I should be thinking about the layers, but I don't see the battery or solar panels fitting in any of the other layers.

Reply:
We definitely need a new layer.

I'll get back to you on how we should think of layers moving forward.